### PR TITLE
Organization Swagger

### PR DESCRIPTION
**Pull request by jjennings-va**
Organization API created to generate swagger (Currently all search params are optional)

### CODE REVIEW

File: provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java [hunk 1]
```diff
@@ -1,36 +1,98 @@
...
             @Content(
                     mediaType = "application/json+fhir",
-                    schema = @Schema(implementation = Location.class)
+                    schema = @Schema(implementation = Organization.class)
             )
     )
-    Location LocationRead(
+    @ApiResponse(
+            responseCode = "400",
+            description = "Not found",
```
Comment (jgarnhart-va) [inline provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java]: 400 is bad request.

404 is not found.

File: provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java [hunk 2]
```diff
@@ -1,36 +1,98 @@
...
+            responseCode = "200",
+            description = "Record found",
+            content =
+            @Content(
+                    mediaType = "application/json+fhir",
+                    schema = @Schema(implementation = Organization.Bundle.class)
+            )
+    )
+    @ApiResponse(
+            responseCode = "400",
```
Comment (jgarnhart-va) [inline provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java]: Same here.

File: provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java [hunk 3]
```diff
@@ -1,36 +1,98 @@
...
+            @Content(
+                    mediaType = "application/json+fhir",
+                    schema = @Schema(implementation = OperationOutcome.class)
+            )
+    )
+    Organization.Bundle organizationSearch(
+            @Parameter(in = ParameterIn.QUERY, name = "identifier") Identifier identifier,
+            @Parameter(in = ParameterIn.QUERY, name = "address") String address,
+            @Parameter(in = ParameterIn.QUERY, name = "name") String name,
+            @Parameter(in = ParameterIn.QUERY, name = "code") String code,
```
Comment (jgarnhart-va) [inline provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java]: Don't see query param `code` in the spec.

File: provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java (hunk 1 above)
Comment (jjennings-va) [reply provider-directory-api/src/main/java/gov/va/api/health/providerdirectory/api/OrganizationApi.java]: I agree

**Pull request merged**